vllm/transformers_utils/configs/chameleon.py

```diff
@@ -0,0 +1,98 @@
+from transformers import PretrainedConfig
+
+
+class ChameleonConfig(PretrainedConfig):
```

Comment:
curious, why do we copy over some `PretrainedConfig` while leaving out others? Like for `LlavaConfig`, we just used the one from huggingface.

Reply:
I'll change to import it once `transformer` has a new release with the PR merged, otherwise we still need this copied over for now.

Reply:
Leave a TODO then?

Reply:
Sounds good